executor/distsql.go

```diff
@@ -1050,6 +1050,8 @@ func (w *tableWorker) compareData(ctx context.Context, task *lookupTableTask, ta
 				tp := &col.FieldType
 				idxVal := idxRow.GetDatum(i, tp)
 				tablecodec.TruncateIndexValue(&idxVal, w.idxLookup.index.Columns[i], col.ColumnInfo)
+				idxVal.SetCollation(charset.CollationBin)
```

Comment:
Please add a comment for here?

Reply:
No, we will remove this because there are some tests fails and it needs further investigation.

Reply:
OK.